From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martijn Muijsers <martijnmuijsers@live.nl>
Date: Thu, 1 Dec 2022 17:26:18 +0100
Subject: [PATCH] Use timin.gs by default

License: GPL-3.0 (https://www.gnu.org/licenses/gpl-3.0.html)
Gale - https://galemc.org

This patch is based on the following patch:
"Optimize default values for configs"
By: etil2jz <81570777+etil2jz@users.noreply.github.com>
As part of: Mirai (https://github.com/etil2jz/Mirai)
Licensed under: GPL-3.0 (https://www.gnu.org/licenses/gpl-3.0.html)

diff --git a/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java b/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java
index f2e30d8743a97c0541808dc5c03300c0d881b843..71f1c1de486a72a6253b69b1a6a0a365faa5b5b4 100644
--- a/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java
+++ b/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java
@@ -98,7 +98,7 @@ public class GlobalConfiguration extends ConfigurationPart {
     public class Timings extends ConfigurationPart {
         public boolean enabled = true;
         public boolean verbose = true;
-        public String url = "https://timings.aikar.co/";
+        public String url = "https://timin.gs/"; // Gale - use timin.gs by default
         public boolean serverNamePrivacy = false;
         public List<String> hiddenConfigEntries = List.of(
             "database",
